easybuild/easyconfigs/n/ncurses/ncurses-6.0.eb

```diff
@@ -16,9 +16,10 @@ sources = [SOURCE_TAR_GZ]
 
 configopts = [
     # default build
-    '--with-shared --enable-overwrite',
+    '--with-shared --enable-overwrite --with-termlib',
```

Comment:
OK, thanks for clarifying.

Looking back at the test report at https://gist.github.com/boegel/5c3303ac90ceea87960a213cf33fc024, building `Lua-5.1.4-8.eb` was indeed failing without the split:

```
gcc -o lua  lua.o liblua.a -lm -Wl,-E -ldl -Wl,-Bstatic -lhistory -lreadline -ltinfo -lncurses  -Wl,-Bdynamic
gcc -o luac  luac.o print.o liblua.a -lm -Wl,-E -ldl -Wl,-Bstatic -lhistory -lreadline -ltinfo -lncurses  -Wl,-Bdynamic
/usr/bin/ld: cannot find -ltinfo
/usr/bin/ld: cannot find -ltinfo
```

Maybe the proper fix would have been to not let Lua link with `-ltinfo` though...